client/src/app/notifications/Notifications.less

```diff
@@ -2,7 +2,7 @@
   display: flex;
   flex-direction: column;
 
-  z-index: 500;
+  z-index: 100000;
```

Comment:
How about if we solve it with CSS layers? We can stop z-index bidding today!
https://developer.mozilla.org/en-US/docs/Web/CSS/@layer

Reply:
Hmm, sounds not trivial to me, from a first look. :upside_down_face: 

But probably worth to investigate if / how layers can help, as part of a :zap:

Reply:
Yeah, we need dedicated time to solve this.

Reply:
I've added this a slack time idea: https://github.com/bpmn-io/internal-docs/pull/1164

Reply:
TIL about CSS layers. 🤯